src/components/ContactUs/ContactUs.tsx

```diff
@@ -0,0 +1,28 @@
+import React, { useEffect, useState } from 'react';
+import DesktopVersion from './DesktopVersion';
+import MobileVersion from './MobileVersion';
+
+export default function ContactUs() {
+  const [screenWidth, setScreenWidth] = useState(1024);
```

Comment:
Because of this initial value, we have a flash of the desktop version on mobile before the Effect runs and sets the correct screenWidth. It would be better to display nothing before we get the dimensions from the browser.

I suggest we initialize this with `null` and return nothing if it is null.

```tsx
const [screenWidth, setScreenWidth] = useState<number | null>(null);

// the useEffect

if (!screenWidth) return null;

// return the JSX
```

Reply:
Nice catch. Fixed via [c1208b8](https://github.com/reactdeveloperske/reactdevske-website/pull/102/commits/c1208b8052fffb34b92ef71fcb5852d18f70fa39)